website/client/components/auth/registerLoginReset.vue

```diff
@@ -214,6 +223,15 @@
       }
     }
 
+    .social-button + .social-button {
```

Comment:
isn this a duplicate of line 132?

Reply:
The css selector on line 132 is for the media queries of a screen and a max width of 768px, while line 226 is for bigger screens.

Reply:
ah sorry, that's ok then!